src/Meta/Utils.cpp

```diff
@@ -117,16 +117,20 @@ bool isAlpha(const std::vector<std::string>& strings, size_t index)
     return true;
 }
 
-std::string Utils::getCommonWordPrefix(const std::vector<std::string>& strings)
+std::string Utils::calculateEnumFieldsPrefix(std::string enumName, std::vector<std::string> fields)
```

Comment:
Ooops, I merged the pull request without checkoing for notes. I will change `enumName`   argument to be `const`. `fields` argument is not `const` because it is changed in method body (`push_back` is called). `push_back` can be called before calling `calculateEnumFieldsPrefix` method in order to avoid changing `fields` argument, but I think this operation is specific to `calculateEnumFieldsPrefix` method and it should happen in the method body.